README.md

```diff
@@ -87,9 +59,8 @@ As an example, you may want to be able to make calls to `sendDocument` with an o
 (Your code is then assumed to able to translate calls to `sendDocument` and the like to multipart/form-data uploads when supplied with an object alike `{ path: '/tmp/file.txt' }` in the `document` property of the argument object.)
 
 `typegram` cannot possibly know what objects you want to support as `InputFile`s.
-Consequently, the exposed type `InputFile` is merely an alias for `never`.
 
-However, you can specify your own version of what an `InputFile` is, hence effectively creating a completely new version of `typegram` with your custom `InputFile` type used throughout all affected methods and interfaces.
+However, you can specify your own version of what an `InputFile` is throughout all affected methods and interfaces.
 This is possible by what we call a _proxy type_.
```

Comment:
Proxy types are no longer used. It now is just a regular type param.

Reply:
Removed reference to proxy type.

Reply:
Thanks